legal-api/src/legal_api/resources/v2/request_tracker.py

```diff
@@ -67,14 +70,14 @@ def resubmit_bn_request(identifier: str):
         request_type=request_type,
         retry_number=-1,
         service_name=RequestTracker.ServiceName.BN_HUB,
-        legal_entity_id=legal_entity.id,
+        legal_entity_id=business.id,
```

Comment:
this will need to be updated to support SPs.  SPs will use `alternate_name_id` in `RequestTracker` model instead of `legal_entity_id`

Reply:
updated